python/jobs.py

```diff
@@ -446,6 +446,7 @@ class Assignment(sdk.python.entities.Entity):
     production_deadline = Property(datetime.datetime)
     assignment_deadline = Property(datetime.datetime)
     job = Property(Job, backref="assignments")
+    supply_job = Property(Job, backref="supply_assignment")
```

Comment:
I feel like you meant to put this on Assignment model.

A job has a job named job, and also a job has a job named assignment? Doesn't seem right.

Reply:
Yeah I think @DanielCollins is right.

Reply:
On second look it is on the Assignment model. The Assignment model is defined at the bottom of the jobs.py file; maybe it's time to move Assignments into it's own file?

Reply:
that seems like a good idea

Reply:
https://github.com/merchisdk/sdk/issues/145